GoedeDoelenHelpen/ClientApp/src/app/user/dashboard/likes-chart/likes-chart.component.ts

```diff
@@ -0,0 +1,123 @@
+import { Component, OnInit } from '@angular/core';
+import { ChartSettings } from '../dashboard.component';
+
+@Component({
+  selector: 'app-likes-chart',
+  templateUrl: './likes-chart.component.html',
+  styleUrls: ['./likes-chart.component.scss']
+})
+export class LikesChartComponent implements OnInit {
+  cs = new ChartSettings();
+
+  chartType = 'bar-vertical-stacked';
+  view: any[];
+  legendTitle = '';
+  xAxisLabel = 'Maand';
+  yAxisLabel = 'Likes';
+
+  scheme = this.cs.getColorScheme(this.cs.colorScheme);
+  schemeType = 'ordinal';
+
+  results = [
```

Comment:
Het is volgens mij de convention de values te definiëren in de constructor in plaats van bovenin de class.
Ook zou ik results verplaatsen naar ngOnInit zodat de data alleen geladen wordt als de rest in plaats is.

Reply:
Het is maar voorbeelddata totdat ik data uit de backend krijg